src/filters.js

```diff
@@ -119,10 +119,12 @@ var filters = {
         if(str instanceof r.SafeString) {
             return str;
         }
+        str = (str === null || str === undefined) ? '' : str;
```

Comment:
maybe another PR.  This one is focus at quick bug fix.
I'm afraid it will cause BREAK CHANGE.